### PR TITLE
Update WalletStoreType according name convention #4027

### LINKED ISSUE #4027: Update WalletStoreType according name convention

### PR DESCRIPTION
**Pull request by oa-s**
Closes #4027

### CODE REVIEW

File: AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/WalletConnect/WalletConnectSwiftV2",
         "state": {
           "branch": null,
-          "revision": "784eafb6c40965addf6d4c0ddee70ad413de4c84",
-          "version": "0.4.0"
+          "revision": "5ea0265d34f18ec48d71fc299adacecdacc904bb",
```
Comment (hboon) [inline AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:9]: This seems unrelated?

**Review by hboon — APPROVED** [review]
Help fix the tests before merging?

File: AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (oa-s) [reply AlphaWallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:9]: yeah, checking
